src/Symfony/Component/HttpKernel/Fragment/InlineFragmentRenderer.php

```diff
@@ -119,7 +119,13 @@ protected function createSubRequest($uri, Request $request)
         // Sub-request object will point to localhost as client ip and real client ip
         // will be included into trusted header for client ip
         try {
-            if ($trustedHeaderName = Request::getTrustedHeaderName(Request::HEADER_CLIENT_IP, false)) {
+            $hasTrustedHeaderSet = method_exists(Request::class, 'getTrustedHeaderSet');
```

Comment:
I would avoid this check by bumping the min version of HttpFoundation in HttpKernel